### ISSUE TITLE
Investigate onminus notes in Conneres cmake configuration.

### ISSUE THREAD

**Issue by ZelRai** [ZeldaClassic/ZeldaClassic#9]
# The sound system has a known memory issue, so for now set the default debugging args to disable sound.
	set_property(TARGET ${zc_targets} PROPERTY VS_DEBUGGER_COMMAND_ARGUMENTS "$<$<CONFIG:ASAN>:-s>")

# REQUIRED or else MSVC throws internal compiler error(!)
	add_compile_options("/fp:fast")
elseif(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
	# TODO: GCC w/ these flags don't quite match the behavior of MSVC or clang, as seen in a replay report:
	#    https://zc-replay-compare-3700187020.surge.sh/
	# The difference is trivial (and isolated to just stuff re: Patras, or maybe Sin/Cos, unclear), and actually GCC
	# looks more "correct". Should re-assess fp-rounding to guarentee same playback for any compiler; should either:
	# 1) ignore some frames for purposes of replay assert
	# 2) investigate more how to get exact same behavior from gcc
	# 3) investigate removing the fp flags from all compilers and seeing if results can be made to be consistent;
	#    would require a workaround for the MSVC internal compiler error mentioned above.
	#
	# In the meantime, we just use clang or msvc (never gcc) for replay tests (and for building release), which seems fine long term.
# Optimize the math functions always, even for debug mode. This reduces many inconsistencies with replays
# in non-Release builds.
if(MSVC)
	# error D8016: '/O2' and '/RTC1' command-line options are incompatible
	# set_source_files_properties(src/base/zc_math.cpp PROPERTIES COMPILE_FLAGS /O2)
else()
	set_source_files_properties(src/base/zc_math.cpp PROPERTIES COMPILE_FLAGS -O3)
endif()

Short version:

1. Sound should not be discluded from producing debug symbols. This is a bandaid not a solution. I don't know who's brainchild this was, but I can guess.

2. We need to figure out how to make the program issue consistant behavior without using optimizations.  This is just poor practice on some one's part, again I can guess.

**Comment by ZelRai** [comment]
The test quests and replays are being stored elsewhere due to storage limitataions. We are not subscribing to Github premium services at this time.
